ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/wan.py

```diff
@@ -296,17 +296,19 @@ def filtered_wan_route_servers(self: SharedUtils) -> dict:
             if wan_rs == self.hostname:
                 # Don't add yourself
                 continue
-
+            ssl_profile = ""
```

Comment:
Since you set it in both the `if` and `else` case below, this should not be needed.